lib/libc/string/bstring.3

```diff
@@ -81,9 +81,9 @@ See the specific manual pages for more information.
 .Xr bcopy 3 ,
 .Xr bzero 3 ,
 .Xr memccpy 3 ,
+.Xr memccpy 3 ,
```

Comment:
Good catch!  ... huh, `main` currently has:
```
.Xr memccpy 3 ,
.Xr memchr 3 ,
.Xr memcmp 3 ,
.Xr memccpy 3 ,
.Xr memcpy 3 ,
```

(with two `memccpy`)

I'll switch the script to use a `set()` instead of a list, so that it eliminates any duplicate lines.  Give me a few minutes.